static/js/click_handlers.js

```diff
@@ -469,6 +469,10 @@ exports.initialize = function () {
         }, 300);
     });
 
+    $("#settings_page").on("click", ".collapse-settings-btn", function () {
+        var elem = $(".collapse-org-settings");
```

Comment:
I think it'd be more readable to not have this as an argument to the function, and instead just do _.each($(".collapse-org-settings"), function(elem) {` in the other file.  (We generally use that style, without `this`, for better readability).

Reply:
That definitely makes it a lot more readable. Made the change.